### PR TITLE
Fix style lint and Update Readme file

### PR DESCRIPTION
**Pull request by asimashfaq**
1. Add node_modules to .stylelintignore to remove linting warnings.
2. Add the Postgresql options in the Readme file.

### CODE REVIEW

File: .stylelintignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 public/scss/_variables.scss
 public/scss/_variables_reference.scss
+node_modules
```
Comment (chappjc) [inline .stylelintignore]: Please make a comment about this change in the PR so I can include that with the merge commit message.

**Comment by chappjc** [comment]
Original PR (with review) that was accidentally closed: https://github.com/decred/dcrdata/pull/1557

File: README.md [hunk 2]
```diff
@@ -372,6 +372,15 @@ cp sample-dcrdata.conf ~/.dcrdata/dcrdata.conf
 Then edit dcrdata.conf with your dcrd RPC settings. See the output of `dcrdata
 --help` for a list of all options and their default values.
 
+### Edit postgresql settings in drcdata.conf file.
```
Comment (chappjc) [inline README.md]: I'm still uncertain about including this (there are a lot of dcrdata.conf settings that should be reviewd before running), but not describing the `psql` commands (`CREATE USER...`, `CREATE DATABASE...`, etc.) that were the initial hang up.

Comment (asimashfaq) [reply README.md]: got it

**Comment by chappjc** [comment]
No changes for about 30 days, so closing.  The PR may be reopened if changes are planned.

**Pull request closed without merging**